.github/workflows/bcny-firebase.yml

```diff
@@ -52,3 +52,37 @@ jobs:
         with:
           name: firebase-windows-amd64
           path: ${{ github.workspace }}/BuildRoot/Library/firebase
+
+      - run: |
+          echo build_tag=$(date +%Y%m%d.$(date +%-H/6 | bc)) >> ${GITHUB_ENV}
+
+      - uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        id: create_release
+        with:
+          draft: false
+          prerelease: false
+          release_name: ${{ env.build_tag }}
+          tag_name: ${{ env.build_tag }}
+
+      - name: nuget package
+        run: |
+          $treeish = git log -1 --format=%h
+          nuget pack -Properties DESTDIR=${{ github.workspace }}/BuildRoot/Library/firebase -Version 0.0.0.0-${treeish} ${{ github.workspace }}/SourceCache/firebase-cpp-sdk/firebase.nuspec
+        shell: pwsh
+
+      - name: publish nuget
+        env:
+          GITHUB_TOKEN: ${{ secrets.NUGET_TOKEN }}
+          GITHUB_USERNAME: thebrowsercompany-bot2
```

Comment:
I guess this isn't something we're hoping to upstream?

Reply:
Nope, I see no reason we would ever upstream the nuget package.  If anything, we might consider switching to the usptream package if we could.